### PR TITLE
Upgrade to HttpConfig 0.1.4

### PR DESCRIPTION
**Pull request by lclarksl**

### CODE REVIEW

**Comment by ianlovell** [comment]
:shipit:

File: Sessions/SessionsRepository.fs [hunk 1]
```diff
@@ -25,13 +25,13 @@ let entityToSession (entity : SessionEntity) : Session =
       Status = entity.Status
       Date =
         if entity.Date.HasValue then
```
Comment (jebbinSCL) [inline Sessions/SessionsRepository.fs]: This if statement can be replaced with `Option.ofNullable entity.Date`

File: Sessions/SessionsRepository.fs [hunk 2]
```diff
@@ -25,13 +25,13 @@ let entityToSession (entity : SessionEntity) : Session =
...
-            convertToISO8601 entity.Date.Value
+            Some entity.Date.Value
```
Comment (jebbinSCL) [inline Sessions/SessionsRepository.fs]: This if expression can be replaced with `Option.ofNullable entity.Date`

File: Sessions/SessionsRepository.fs [hunk 3]
```diff
@@ -43,18 +43,16 @@ let sessionToEntity (session : Session) : SessionEntity =
       Status = session.Status
       Date =
         match session.Date with
-        | null -> Nullable()
-        | "" -> Nullable()
-        | _ -> Nullable(convertToDateTime session.Date)
+        | None -> Nullable()
```
Comment (jebbinSCL) [inline Sessions/SessionsRepository.fs]: This match expression can be replaced with `session.Date |> Option.toNullable`

**Comment by pallenSL** [comment]
Put that squirrel on a :boat:   :D

**Comment by lclarksl** [comment]
Thanks for the pointers @jebbinSCL - this way looks much nicer.

**Comment by jebbinSCL** [comment]
No problem!
:ship:

**Pull request merged**